src/stb_image.rs

```diff
@@ -1,181 +1,15153 @@
-// Copyright 2013 The Servo Project Developers. See the COPYRIGHT
```

Comment:
It's not clear to me that committing this bindgen-generated file is an improvement, since it contains a huge amount of macOS-specific types.

Reply:
I agree that there are many OS-specific types in there, but that's what `bindgen` is spitting out. Is there a way to make it avoid pulling in all of the system specific types?

Reply:
Rather than regenerating stb_image.rs, why don't we take the existing one and add a cargo feature for libc support. Then we can conditionally expose the APIs that involve libc::FILE based on that feature, and we should achieve the same effect.

Reply:
I went even further and completely removed the bindings from the repo and let rust-bindings do its job during compile time, which is the current way to use the bindgen.
I also added a test to check that the library is actually working.

Reply:
I've updated this to reflect the current code in master